### PR TITLE
PR template updated.

### PR DESCRIPTION
**Pull request by janosmurai**
## Type Of Change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Any modification in the test cases
- [x] Any modification in the documentation

### Checklist:

* [x] I have read and followed the [contribution guideline](https://github.com/axem-solutions/.github/blob/main/CONTRIBUTING.md).
* [x] I have checked to ensure there aren't other open Pull Requests for the same update/change.
* [ ] Every new modification in the production code has been tested if there is any.
* [ ] New and existing test cases pass.
* [ ] 100% unit test coverage achieved for every production code modification I made.
* [x] I have made corresponding changes to the documentation.

## Related Issue
Closing: 

## Description
PR template updated to better align with the DEM workflow.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,39 +1,42 @@
...
 
 * [ ] I have read and followed the [contribution guideline](https://github.com/axem-solutions/.github/blob/main/CONTRIBUTING.md).
-* [ ] I have checked to ensure there aren't other open Pull Requests for the same update/change
-* [ ] I have tested these changes and added tests that prove my fix is effective or that my feature works
-* [ ] I have made corresponding changes to the documentation
-* [ ] New and existing unit tests pass locally with my changes
-* [ ] 100% unit test coverage achieved
-
+* [ ] I have checked to ensure there aren't other open Pull Requests for the same update/change.
+* [ ] Every new modification in the production code has been tested if there is any.
```
Comment (vezdeneszter) [inline .github/pull_request_template.md:13]: I'm not sure I understand this sentence

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,39 +1,42 @@
...
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Any modification in the test cases
+- [ ] Any modification in the documentation
+
 ### Checklist:
 
 * [ ] I have read and followed the [contribution guideline](https://github.com/axem-solutions/.github/blob/main/CONTRIBUTING.md).
-* [ ] I have checked to ensure there aren't other open Pull Requests for the same update/change
-* [ ] I have tested these changes and added tests that prove my fix is effective or that my feature works
```
Comment (vezdeneszter) [inline .github/pull_request_template.md:5]: This should be kept

**Review by vezdeneszter — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (janosmurai) [reply .github/pull_request_template.md:13]: I can rephrase it like:
If there is any modification in the production code it has been tested.

File: .github/pull_request_template.md (hunk 2 above)
Comment (janosmurai) [reply .github/pull_request_template.md:5]: The first one is kept, the second one has been rephrased.

**Review by vezdeneszter — APPROVED** [review]

**Review by vezdeneszter — APPROVED** [review]

**Pull request merged**